jason-lab/test/user-route.test.js

```diff
@@ -0,0 +1,115 @@
+const app = require('../server');
+const request = require('supertest')(app);
+const {expect} = require('chai');
+
+const User = require('../model/user');
+
+describe('userroutes', function (){
+  describe('POST /api/user', function (){
+    describe('with a body', function (){
+      after(function (){
+        return User.remove(this.testUser);
```

Comment:
`this.testUser` isn't set in this context

Reply:
I"M GONNA DELETE IT ANYWAY